roles/bootstrap-os/tasks/bootstrap-centos.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+- name: Check presence of fastestmirror.conf
+  stat: path=/etc/yum/pluginconf.d/fastestmirror.conf
+  register: fastestmirror
+
+# fastestmirror plugin actually slows down Ansible deployments
+- name: Disable fastestmirror plugin
+  lineinfile:
+    dest: /etc/yum/pluginconf.d/fastestmirror.conf
+    regexp: "^enabled=.*"
+    line: "enabled=0"
+    state: present
```

Comment:
Hmm my comment on this got removed by github due to the force push...
I added a conditional to the task so that it is only executed when the file exists. Creating the file with just this line would result in an invalid INI file.